app/code/Magento/Search/Ui/Component/Listing/Column/Website/Options.php

```diff
@@ -32,6 +37,11 @@ class Options implements OptionSourceInterface
      */
     protected $systemStore;
 
+    /**
+     * @var array
+     */
+    protected $options;
```

Comment:
Looks like in this class there options variable is only written and never used for read, maybe it's better to remove writing to it?

Reply:
Understood, but for backward compatibility this might be needed in case someone has a preference this and is expecting the value in the mentioned field.